docs/en-US/component/date-picker.md

```diff
@@ -141,7 +141,7 @@ Note, date time locale (month name, first day of the week ...) are also configur
 
 | Attribute             | Description                                                                                           | Type                                             | Accepted Values                                                                                                   | Default     |
 | --------------------- | ----------------------------------------------------------------------------------------------------- | ------------------------------------------------ | ----------------------------------------------------------------------------------------------------------------- | ----------- |
-| model-value / v-model | binding value                                                                                         | date(DatePicker) / array(DateRangePicker)        | —                                                                                                                 | —           |
+| model-value / v-model | binding value                                                                                         | date(DatePicker) / array(DateRangePicker)        | Array with length 2                                                                                               | —           |
```

Comment:
It may be possible to omit the description in the `Accepted Values` and modify the `Type` to `Date / [Date, Date]`